packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -64,6 +24,194 @@ export enum AttachState {
     Detached = "Detached"
 }
 
+// @alpha
```

Comment:
I think everything in this file needs to be `@public` to ensure we have a clear support promise (these are part of the API surface we intend you to use), otherwise it's kind of muddy.  @DLehenbauer WDYT?

Or else we probably need something concrete about what "exported but `@alpha`" means -- available but no breaking change promise?  available with full support promise?

Reply:
I don't think our policy with respect to public vs alpha stuff is specific to this package, so I don't think this is the place to do that. I have separate work to document what we mean by supported and breaking changes, which can cover things like relying on undocumented behavior or non-public API not being supported and thus not being covered by our no breaking changes policy and what ever more formal policy we have for support.

Reply:
I also don't see this PR as making any changes to what we support. For example before this PR IFluidHandle was referenced, and changing it could break users of this package. That is still true, users just don't have to depend on another package if they want to save a handle in a type variable.

Reply:
@ChumpChief - The `@alpha` tag is how we get Fluid Framework 1.0 types to show up under the 'fluid-framework/legacy' path.  (It's the same as how we use `@alpha` elsewhere.)

The message to customers is that '/legacy' types are supported, but not recommended for new designs.  We'll mark the types as `@deprecated` when we have a way for customers to migrate their data forward to SharedTree.